### ISSUE TITLE
[bug]: bunx --bun shadcn-ui@latest add table

### ISSUE THREAD

**Issue by duran-0519** [shadcn-ui/ui#3682]
### Describe the bug

ConnectionClosed: The socket connection was closed unexpectedly. For more information, pass `verbose: true` in the second argument to fetch()
 path: "https://ui.shadcn.com/registry/index.json"

Failed to fetch components from registry.

### Affected component/components

table

### How to reproduce

bunx --bun shadcn-ui@latest add table
FetchError: request to https://ui.shadcn.com/registry/index.json failed, reason: read ECONNRESET
    at ClientRequest.<anonymous> (file:///private/tmp/bunx-501-shadcn-ui@latest/node_modules/node-fetch/src/index.js:108:11)
    at ClientRequest.emit (node:events:514:28)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  type: 'system',
  errno: 'ECONNRESET',
  code: 'ECONNRESET',
  erroredSysCall: 'read'
}
Failed to fetch components from registry.

### System Info

```bash
macOS Monterey 12.7.4
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by dxl519** [comment]
node -v
v20.10.0
bun -v
1.1.4

**Comment by lubosmato** [comment]
I got similar error. I accidentally ran `yarn install` in my bun project folder which added `"packageManager": "..."` into `package.json` and forgot about it. 

This confused package manager detection and `bunx --bun shadcn-ui@latest add ...` command then uses yarn package manager.

I fixed it by removing all yarn mentions (yarn.lock, packageManager in package.json, ...) and re-run `bunx --bun shadcn-ui@latest add ...`.
